app/services/provider_notifications.rb

```diff
@@ -4,5 +4,10 @@ def send_provider_created_notifications(linked_token)
       ProviderMailer.with(linked_token: linked_token).request_created_email.deliver_later
       InternalMailer.with(linked_token: linked_token).request_created_email.deliver_later
     end
+
+    def send_volunteer_response_created_notifications(provider, volunteer, response)
```

Comment:
This is great! Minor request: can we rename this class to `Notifications` or create a `VolunteerNotifications` class?

Reply:
On second thought, this makes sense as is :) Ignore the last comment